src/main.js

```diff
@@ -236,10 +236,13 @@ app.on('ready', function() {
       mainWindow.focus();
     });
     ipcMain.on('notified', function(event, arg) {
-      if (process.platform === 'win32') {
+      if (process.platform === 'win32' || process.platform == 'linux') {
```

Comment:
No particular reason :wink: Should not be a problem semantically.
But you are right, it does not match the surrounding code style, will be updated.